data/course.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "Your course title",
+  "description": "Your course description",
+  "version": "0.1.0",
```

Comment:
we added a version to the courses, which gives us this parameter?

Reply:
The capability to versioning the content based on this param (in a "on fly" mode we wouldn't have a package.json file where we are used to).